.golangci.yml

```diff
@@ -21,3 +21,6 @@ run:
   deadline: 2m
   skip-files:
     - "zz_generated.*\\.go$"
+  skip-dirs:
+    - src/external_libs
+    - controllers/drain
```

Comment:
It might be good to move the drain implementation copied from upstream to a different directory to help make it clear that we don't truly "own" these files.